app/src/main/java/de/xikolo/controllers/channels/ChannelDetailsActivity.kt

```diff
@@ -0,0 +1,112 @@
+package de.xikolo.controllers.channels
+
+import android.os.Bundle
+import android.view.Menu
+import android.view.MenuItem
+import android.view.View
+import android.view.ViewStub
+import android.widget.ImageView
+import androidx.core.app.NavUtils
+import butterknife.BindView
+import com.google.android.material.appbar.AppBarLayout
+import com.google.android.material.appbar.CollapsingToolbarLayout
+import com.yatatsu.autobundle.AutoBundleField
+import de.xikolo.R
+import de.xikolo.config.GlideApp
+import de.xikolo.controllers.base.BaseActivity
+import de.xikolo.controllers.helper.CollapsingToolbarHelper
+import de.xikolo.models.dao.ChannelDao
+import de.xikolo.utils.ShareUtil
+
+class ChannelDetailsActivity : BaseActivity() {
+
+    companion object {
+        val TAG: String = ChannelDetailsActivity::class.java.simpleName
+    }
+
+    @AutoBundleField
+    lateinit var channelId: String
+
+    //-1 do not scroll to Course
+    @AutoBundleField(required = false)
+    var scrollToCoursePosition = -1
+
+    @BindView(R.id.toolbar_image)
+    lateinit var imageView: ImageView
+
+    @BindView(R.id.appbar)
+    lateinit var appBarLayout: AppBarLayout
+
+    @BindView(R.id.collapsing_toolbar)
+    lateinit var collapsingToolbar: CollapsingToolbarLayout
+
+    @BindView(R.id.stub_bottom)
+    lateinit var stubBottom: ViewStub
```

Comment:
This variable is unused, should it be removed?

Reply:
Done.